### DISCUSSION TITLE
How to view the complete knowledge graph generated by Graphrag

### DISCUSSION THREAD

**Discussion by niubiqianshui** [Azure-Samples/graphrag-accelerator#76] [Q&A] [answered]
How can I view the knowledge graph corresponding to a text file that I added to Graphrag using the index command

**Comment by timothymeyers** [comment, marked as answer]
@niubiqianshui - see the section titled "Exploring the Knowledge Graph" in https://github.com/Azure-Samples/graphrag-accelerator/blob/main/notebooks/2-Advanced_Getting_Started.ipynb. Specifically, you can download a `graphml` file from the API and that can be imported into visualization software. We use [Gephi](https://gephi.org/).

There's a number of issues and discussions about this in the GraphRAG Python library repo, e.g. - https://github.com/microsoft/graphrag/issues/418 and https://github.com/microsoft/graphrag/issues/446#issuecomment-2218678884

We might create a guide in the future that walks through the process we use with Gephi.
